Package commun

florianed opened the pull request:
Nouvelles interfaces

Commit:
modif Lindt

Commit:
Merge branch 'master' of https://github.com/florianed/CACAO2016.git

maylinh committed:
modification de tréso en cours, sauvegarde

maylinh committed:
Merge branch 'master' of https://github.com/florianed/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe5/Tresorerie.java

maylinh committed:
cout de stock et de livraison

florianed committed:
Création des stocks des 3 chocolats

florianed committed:
Rajouts dans le next

florianed committed:
Suppression de l'indicateur stockChocolat

florianed committed:
J'ai enlevé certains doublons

maylinh committed:
interfaces et commande distri

maylinh committed:
Merge branch 'master' of https://github.com/florianed/CACAO2016.git

florianed committed:
Préparation du fetch

florianed committed:
Merge branch 'master' of https://github.com/florianed/CACAO2016.git

florianed committed:
fetch pour tout le monde

Commit:
…

Commit:
Merge branch 'master' of https://github.com/florianed/CACAO2016.git

florianed committed:
rien

massimotisi commented:
Je ne peux pas l'accepter. Pour le changement d'interface je suggere:
1- d'ajouter la nouvelle version de la methode (en choisissant un nouveau nom si necessaire)
2- d'ajouter l'annotation @deprecated à l'ancienne version

L'ancienne methode continue à fonctionner, en laissant le temps à tout le monde d'adapter le code à la nouvelle interface. 
Toutes les methodes @deprecated seront supprimé le jour de la release 2.0

Commit:
modif Lindt

florianed committed:
Merge branch 'master' of https://github.com/florianed/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/ITransformateur.java
	PRIME_CACAO_2016/src/abstraction/equipe5/Lindt.java

maylinh committed:
ajout du prix dans commande distri

florianed committed:
Merge branch 'master' of https://github.com/florianed/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe5/Lindt.java

florianed committed:
Message de romu : il y a des problemes pour les autres equipes du fait
qu'il y a des nouvelles methodes qu'ils n'ont pas encore implementees
--> il leur suffit de faire "add unimplemented methods" pour que ca
marche... on pourrait le faire pour eux mais chaque equipe ne peut pas
toucher au package des autres.
(c'est vraiment romu qui ecrit ça :) )

florianed commented:
Nouvelles pull request

maylinh committed:
création venteDist

florianed committed:
AchatProd

maylinh committed:
Merge branch 'master' of https://github.com/florianed/CACAO2016.git

Conflicts:
	PRIME_CACAO_2016/src/abstraction/equipe5/Lindt.java

florianed committed:
Merge branch 'master' of https://github.com/florianed/CACAO2016.git

massimotisi commented:
Pouvez-vous ajouter les methodes qui manquent (vides pour l'instant) dans les autres packages avec la benediction de @jhtr, @AlexandreMarty, @MarcSuteau, @SofianeSalort et @lakac ?

florianed committed:
Ajout des méthodes

florianed commented:
Voilà, c'est fait.

massimotisi commented:
Votre pull request est en conflit avec ma version, avez-vous synchronisé avec mon repository?

massimotisi commented:
Votre pull request est encore en conflit, je ne peux pas merger.

florianed committed:
Merge remote-tracking branch 'choose_remote_name/master'

Conflicts:
	PRIME_CACAO_2016/src/abstraction/commun/ITransformateur.java
	PRIME_CACAO_2016/src/abstraction/equipe2/Transformateur2.java
	PRIME_CACAO_2016/src/abstraction/equipe5/Lindt.java

florianed commented:
Et maintenant, est-ce que cela fonctionne?

massimotisi commented:
Non, il ne te dit pas sur cette page que ta version est en conflit?

massimotisi commented:
Il faut que tu re synchronise avec mon repository (fetch from..., merge)

florianed committed:
Nouveau commit

florianed committed:
Nouveau commit
